### DISCUSSION TITLE
Brainstrom on compare features

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/awesome-github-profiles#702] [Ideas]
here is the link: https://recodehive.github.io/awesome-github-profiles/pages/compare.html

**Comment by Jisha-tr** [comment]
i think we should change the theme

**Comment by Peart-Guy** [comment]
> here is the link: https://recodehive.github.io/awesome-github-profiles/pages/compare.html

@sanjay-kv  I can work on this 
Please  assign me
I have also raised an issue against this #863 
Name : Ankan
GitHub Name : Peart-Guy
Participant role : GSSOC-24 EXTD , Hacktoberfest
